assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+		"LILAC": ["N34d27m10.70",	"E134d58m42.00"],
+		"MAIKO": ["N34d36m39.70",	"E134d59m49.10"],
+		"MAYAH": ["N34d36m23.70",	"E135d12m11.00"],
+		"NALTO": ["N34d09m00.60",	"E134d43m13.30"],
+		"NARAH": ["N34d29m58.80",	"E135d41m02.90"],
+		"ODORI": ["N34d00m13.20",	"E134d30m11.80"],
+		"SHION": ["N34d25m42.10",	"E134d16m57.40"],
+		"SHTLE": ["N34d49m51.00",	"E136d56m53.80"],
+		"SOUJA": ["N34d37m38.60",	"E133d44m22.50"],
+		"TANTA": ["N34d16m04.90",	"E134d27m29.20"],
+		"TOMOH": ["N34d16m49.60",	"E135d00m20.30"],
+		"TONDA": ["N33d43m43.80",	"E135d22m39.30"],
+		"UENOH": ["N34d50m08.70",	"E136d13m33.70"],
+		"WASYU": ["N34d28m17.50",	"E133d23m01.10"],
+		"WENDY": ["N34d32m09.50",	"E134d50m30.80"],
+		"YAGYU": ["N34d32m39.30",	"E135d57m38.10"],
+		"YME"  : ["N35d28m50.50",	"E135d08m13.30"],
+		"YOSAK": ["N33d47m04.70",	"E134d06m38.40"],
+		"HAMAR": ["N34d24m34.70",	"E135d03m04.10"],
+		"KCC"  : ["N35d15m55.00",	"E136d54m53.70"],
+		"KCE"  : ["N34d37m52.00",	"E135d13m42.00"],
+		"KNE"  : ["N34d25m48.00",	"E135d15m06.00"],
+		"KTE"  : ["N34d12m45.00",	"E134d01m21.00"],
+		"KYOTO": ["N34d49m49.70",	"E135d44m48.80"],
+		"LINDA": ["N34d30m15.80",	"E134d51m40.10"],
+		"NANKO": ["N34d36m38.70",	"E135d21m21.00"],
+		"OGURA": ["N34d47m58.70",	"E135d34m24.90"],
+		"OYE"  : ["N34d45m01.00",	"E133d50m06.00"],
+		"SIGAK": ["N34d53m07.70",	"E136d02m52.70"],
+		"TSC"  : ["N34d07m47.00",	"E134d36m31.00"],
+		"UMAJI": ["N33d45m00.50",	"E134d09m00.60"],
+		"BRAVE": ["N33d25m50.00",	"E133d33m30.00"],
+		"FUE":   ["N32d40m07.00",	"E128d49m36.00"],
+		"KARIN": ["N33d40m40.00",	"E134d11m50.00"],
+		"KISEI": ["N33d28m04.00",	"E135d19m44.00"],
+		"OOITA": ["N33d13m13.00",	"E131d42m12.00"],
+		"RANDY": ["N34d06m35.00",	"E133d53m51.00"],
+		"SALTY": ["N33d51m10.00",	"E132d55m31.00"],
+		"STORK": ["N32d56m48.00",	"E133d14m35.00"],
+		"SUC":   ["N32d45m21.00",	"E132d59m48.00"],
+		"TAPOP": ["N32d40m01.00",	"E136d07m57.00"]
+	},
+    "runways": [{
+        "name": ["06L", "24R"],
+        "name_offset": [
+            [0, 0], [0, 0]
+        ],
+        "end": [
+            ["N34d25m42.86", "E135d12m22.33", "122.5ft"],
+            ["N34d27m04.58", "E135d14m24.08", "122.6ft"]
+        ],
+        "length": 4.000,
+        "ils": [true, true]
+    }, {
+        "name": ["06R", "24L"],
+        "name_offset": [
+            [0, 0], [0, 0]
+        ],
+        "end": [
+            ["N34d25m02.56", "E135d13m45.40", "122.8ft"],
+            ["N34d26m14.04", "E135d15m31.92", "123.0ft"]
+        ],
+        "length": 3.500,
+        "ils": [true, true]
+    }],
+    "sids": {
+        "MAIKO6": {
+            "icao": "MAIKO6",
+            "name": "Maiko Six",
+            "exitPoints": {
+                "MAIKO": [
+                    ["MAIKO", "A50+"]
+                ]
+            },
+            "rwy": {
+                "06L": ["KNE", ["MAIKO", "A50+"]],
+                "06R": ["KNE", ["KCE", "A25+"], ["MAIKO", "A50+"]],
+                "24L": [],
+                "24R": []
```

Comment:
There will be some issues here:
 - from Runways 24L & 24R, this will be a "one-fix" procedure, which makes the game freak out.
 - the `MAIKO` fix is listed twice, and it should only be listed once. Else, the aircraft will have a waypoint list of `KNE, KCE, MAIKO, MAIKO` from `06R`.